app/Console/Commands/ParseGeomBoundariesProjectsCommand.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\V2\Projects\Project;
+use App\Services\PolygonService;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\App;
+
+class ParseGeomBoundariesProjectsCommand extends Command
+{
+    protected $signature = 'parse:project-boundaries {framework_key}';
+
+    protected $description = 'Parse string geojsons of projects and project pitches by framework_key';
+
+    public function handle()
+    {
+        $frameworkKey = $this->argument('framework_key');
+        $this->info($frameworkKey);
+
+        $projects = Project::where('framework_key', $frameworkKey)
+        ->whereNotNull('boundary_geojson')
+        ->where('boundary_geojson', '!=', 'null')
+        ->get();
+        $bar = $this->output->createProgressBar(count($projects));
```

Comment:
We can't, because it comes from Symfony library.
https://symfony.com/doc/current/console/style.html#progress-bar-methods
https://github.com/symfony/symfony/blob/7.1/src/Symfony/Component/Console/Style/SymfonyStyle.php